setup.py

```diff
@@ -1,6 +1,10 @@
 import versioneer
-
 from setuptools import find_packages, setup
+from pathlib import Path
+
+# Add README.md as long description
+this_directory = Path(__file__).parent
```

Comment:
Is there a reason why we are using `Path(__file__)` here instead of `open("README.md")` like in the next few lines?

Reply:
Just copying the docs from here to be safe: 
https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/